linux.md

```diff
@@ -6,9 +6,9 @@
    ```bash
    curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash -
    sudo apt-get update
-   sudo apt-get install -y git nodejs python3
+   sudo apt-get install -y git nodejs python3 build-essential
```

Comment:
- Is there not an `s` at the end of this package name?
- what is the official name + source of this package? This should also be used in the PR title + description

Reply:
seems like [it is indeed `build-essential`](https://packages.ubuntu.com/kinetic/build-essential) (without `s`)